tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "module": "amd",
     "moduleResolution": "node",
-    "target": "es5",
+    "target": "es6",

```

Comment:
What does changing the target from es5 to es6 do specifically? I did not notice any big differences but I may have overlooked something.